rocketpy/Rocket.py

```diff
@@ -579,20 +590,93 @@ def addFins(
         Ymac = (
             (s / 3) * (Cr + 2 * Ct) / Yr
         )  # span wise position of fin's mean aerodynamic chord
+        gamac = np.arctan((Cr - Ct) / (2 * span))
         Lf = np.sqrt((rootChord / 2 - tipChord / 2) ** 2 + span ** 2)
         radius = self.radius if radius == 0 else radius
         d = 2 * radius
+        Aref = np.pi * radius ** 2
+        AR = 2 * s ** 2 / Af  # Barrowman's convention for fin's aspect ratio
```

Comment:
@MateusStano , wouldn't be better if we make 2*s inside parentesis?
```suggestion
        AR = (2 * s ) ** 2 / Af  # Barrowman's convention for fin's aspect ratio
```

Reply:
No, that would make the AR calculation wrong. It would be 2 times bigger than it should